### PR TITLE
Changes for local booting Debian images

### PR DESCRIPTION
**Pull request by halmartin**
Changes for jessie/stretch to boot using grub-efi and a local kernel

See individual commit messages for more specific description of changes

### CODE REVIEW

File: jessie/overlay/usr/local/sbin/scw-generate-net-config [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
```
Comment (CedricCabessa) [inline jessie/overlay/usr/local/sbin/scw-generate-net-config]: it"s minor but space is not needed in shebang
`#!/bin/bash`

File: jessie/overlay/usr/local/sbin/scw-generate-net-config [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+find /etc/network/interfaces.d/ -not -name "*.user.cfg" -type f -delete
+for iface in $(ls /sys/class/net); do
+    # Only configure physical interfaces
+    if readlink /sys/class/net/$iface | grep --invert-match --silent 'virtual'; then
+        echo """# Generated by $(basename $0)
```
Comment (CedricCabessa) [inline jessie/overlay/usr/local/sbin/scw-generate-net-config]: bash is not python :-)
Here you've create an empty string (the 2 first `"`), then a normal string `# Generated by ....`

File: jessie/overlay/usr/local/sbin/scw-generate-net-config (hunk 1 above)
Comment (halmartin) [reply jessie/overlay/usr/local/sbin/scw-generate-net-config]: I'll make the corrections and let @superseed know, it's originally from his 17.10 overlay ;)

Comment (superseed) [reply jessie/overlay/usr/local/sbin/scw-generate-net-config]: haha we'll need a major release version for this breaking change

File: jessie/overlay/usr/local/sbin/scw-generate-net-config (hunk 2 above)
Comment (superseed) [reply jessie/overlay/usr/local/sbin/scw-generate-net-config]: Yep, seems like too much Python and Groovy made me forget how multiline strings work in bash. (I wrote that)

**Pull request merged**